backend/players/models.py

```diff
@@ -2,19 +2,110 @@
 from django.dispatch import receiver
 from django.contrib.auth.models import User
 from django.db import models
+from django.utils import timezone
 
 
 class Player(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    display_name = models.CharField(max_length=30, unique=True)
-    avatar = models.ImageField(upload_to='avatars/', default='fallback.png', blank=True)
+    user = models.OneToOneField(User, on_delete=models.CASCADE) # Registration information
+    display_name = models.CharField(max_length=30, unique=True) # Name that would be displayed to other players
+    avatar = models.ImageField(upload_to='avatars/', default='fallback.png', blank=True) # Profile picture
+    created_at = models.DateTimeField(default=timezone.now) # Date of account creation
+    updated_at = models.DateTimeField(auto_now=True) # Date of last account update
 
     def __str__(self):
-        return self.display_name
+        return f"Player: {self.display_name}"
 
 # Use signal to remove corresponding user on player deletion
 @receiver(post_delete, sender=Player)
 def delete_user_with_player(sender, instance, **kwargs):
     # Delete the associated user when the player is deleted
     if instance.user:
         instance.user.delete()
+
+
+class Tournament(models.Model):
+    name = models.CharField(max_length=100) # Name of the tournament
+    status = models.CharField(max_length=20, choices=[
+        ('scheduled', 'Scheduled'),
+        ('in_progress', 'In Progress'),
+        ('completed', 'Completed')
+    ], default='scheduled') # Status of the tournament
+    winner = models.ForeignKey(Player,
+                               on_delete=models.SET('Unknown'),
+                               null=True,
+                               blank=True,
+                               related_name='won_tournaments') # Winner of the tournament
+    created_at = models.DateTimeField(auto_now_add=True) # Date of tournament creation
+
+    def __str__(self):
+        return f"Tournament: {self.name} - Status: {self.status}"
+
+# Association table for players and tournaments
+class TournamentParticipant(models.Model):
+    tournament = models.ForeignKey(Tournament,
+                                   on_delete=models.CASCADE,
+                                   related_name='participants') # Tournament the player is participating in
+    player = models.ForeignKey(Player, on_delete=models.CASCADE) # Player participating in the tournament
+    order = models.IntegerField() # Order of the player in the tournament (For seeding purposes)
+
+    class Meta:
+        unique_together = ('tournament', 'player')
```

Comment:
`unique_together` is [considered obsolete](https://docs.djangoproject.com/en/4.1/ref/models/options/#unique-together)
Use [UniqueConstraint](https://docs.djangoproject.com/en/4.1/ref/models/constraints/#django.db.models.UniqueConstraint) with the [constraints](https://docs.djangoproject.com/en/4.1/ref/models/options/#django.db.models.Options.constraints) option instead.

```suggestion
            constraints = [
        models.UniqueConstraint(fields=['tournament', 'player'], name='unique_tournament_participant')
    ]
```